README.md

```diff
@@ -467,6 +467,20 @@ Example, specify ```hci1```:
 sudo NOBLE_HCI_DEVICE_ID=1 node <your file>.js
 ```
 
+If you are using multiple HCI devices in one setup you can run two instances of noble with different binding configurations by initializing them seperatly in code:
+
+```
+const HCIBindings = require('noble/lib/hci-socket/bindings');
+const Noble = require('noble/lib/noble');
+
+const params = {
+  deviceId: 0,
+  userChannel: true
```

Comment:
is this a good name for this?

Reply:
This seems to be the name of the source. So even in the kernel driver it is named like that so I thought I stick with that.